demo/run-tmux.py

```diff
@@ -0,0 +1,188 @@
+''
+# The above quotation is required because we import this
+# module as a nix `String`.
+# Please don't use triple quotes inside this `Python` module.
+
+import libtmux
+from collections import namedtuple
+import os
+import subprocess
+import time
+
+if not os.path.exists('demo'):
+    raise Exception(
+        'Plase navigate to the root directory of the hydra-poc project'
+    )
+
+root_dir = os.getcwd()
+demo_dir = os.path.join(root_dir, 'demo')
+devnet_dir = os.path.join(demo_dir, 'devnet')
+
+os.chdir(demo_dir)
+
+if not os.path.exists(devnet_dir):
+    subprocess.run('./prepare-devnet.sh')
+    subprocess.run('./seed-devnet.sh $(which cardano-cli)')
+
+SESSION_NAME = 'hydra-demo'
+
+
+def await_nseconds(seconds):
+    def _sleep():
+        time.sleep(seconds)
+        return True
+    return _sleep
+
+
+PaneCfg = namedtuple('PaneCfg', ['cmd', 'await_cmd'])
+
+WindowCfg = namedtuple('WindowCfg', ['name', 'panes'])
+
+cardano_node_cmd = ' '.join([
+    'cardano-node run',
+    '--config devnet/cardano-node.json',
+    '--topology devnet/topology.json',
+    '--database-path devnet/db',
+    '--socket-path devnet/ipc/node.socket',
+    '--shelley-operational-certificate devnet/opcert.cert',
+    '--shelley-kes-key devnet/kes.skey',
+    '--shelley-vrf-key devnet/vrf.skey',
+])
+cardano_node_window_cfg = WindowCfg(
+    'cardano-node',
+    [PaneCfg(
+        cardano_node_cmd,
+        await_nseconds(1)
+        # FIXME: We should probably use some more reliable strategy here like:
+        # await_command_succeeds(
+        #    'cardano-cli query tip --testnet-magic 42', 10
+        # )
```

Comment:
IMO this is more a TODO than a FIXME. If it works this way for most users, it's fine.